src/functions.cpp

```diff
@@ -1,18 +1,21 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
-#include <algorithm>
-#include <iostream>
-#include <limits>
-
-void print(std::vector<int> const & numbers) {
-    for (int i = 0; i < numbers.size(); ++i) {
-        std::cout << i << ": " << numbers[i] << '\n';
-    }
-    std::cout << '\n';
-}
-
-void sort(std::vector<int> & unsorted) {
-    std::sort(begin(unsorted), end(unsorted));
-}
```

Comment:
I was wondering why those lines are marked red (like deleted), then I saw that almost every line from the "green" (added) code has a whitespace in front. It is better to delete those whitespaces.